### PR TITLE
amend READMEmd

### PR DESCRIPTION
**Pull request by alessio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,8 +16,71 @@ This repository contains both a library and a standalone binary, named **crg**.
...
+tar -C /usr/local -xzf go1.15.2.linux-amd64.tar.gz
+export PATH=$PATH:/usr/local/go/bin
+```
+
+Get a simple starport chain running in another session
+```bash
+sudo apt install -y screen git build-essential
+screen -S starport
+npm i -g @tendermint/starport
+starport app github.com/yourname/clay
```
Comment (alessio) [inline README.md:46]: Why `clay`?

File: README.md [hunk 2]
```diff
@@ -16,8 +16,71 @@ This repository contains both a library and a standalone binary, named **crg**.
...
+cd clay
+starport serve --verbose
+```
+
+Hold `ctrl` and press A, then D. You'll leave the screen session and the blockchain `clay` will run in the background.
+
+
+Install and use `crd`
```
Comment (alessio) [inline README.md:54]: What is `crd`?

**Comment by faddat** [comment]
re-focus on crg only.

2nd section should describe use as a library

File: README.md (hunk 2 above)
Comment (faddat) [reply README.md:54]: The Camp Road Diner!

Sadly, it no longer exists and is now called Savory's.  Still one of the best breakfasts in my hometown ;).

https://www.savoryshamburg.com/

Comment (alessio) [reply README.md]: LOL

File: README.md (hunk 1 above)
Comment (faddat) [reply README.md:46]: :)

Clay is formless, and can be shaped into anything.

Comment (alessio) [reply README.md:46]: ahhh ok ok :)

**Comment by alessio** [comment]
starts looking beautiful!

**Comment by faddat** [comment]
doing a final pass of checks and will change from draft->PR

**Review by faddat — APPROVED** [review]
looks good

**Review by jgimeno — APPROVED** [review]

**Pull request merged**